### PR TITLE
Blog post about the Cloud Upload

### PR DESCRIPTION
**Pull request by Ana06**
Co-authored-by: @bgeuken

### CODE REVIEW

**Comment by Ana06** [comment]
[P5: (3) 2.9 Cloud Upload Teaser](https://trello.com/c/KilbFTws/682-p5-3-29-cloud-upload-teaser)

File: _posts/development/2018-02-23-cloud-upload.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<img src="/images/posts/cloud_upload_uploader.png" alt="Cloud uploader" style="max-width: 800px;">
+
+If you haven't configured your account yet, you can use the link in the EC2 Uploader for the `Amazon EC2 configuration`.
+Check the [Configuring Amazon EC2 credentials section](#configuring-amazon-ec2-credentials) for more details.
+
+If your account is already configured, you can choose the AMI Name, the region you want to upload to and optionally the VPC Subnet ID.
+
+<img src="/images/posts/cloud_upload_uploader.png" alt="Cloud uploader" style="max-width: 800px;">
+
+By clicking `Upload`, the image process will start and you will get redirected to the `All cloud Uploads` page:
```
Comment (ChrisBr) [inline _posts/development/2018-02-23-cloud-upload.md]: Either ``All Cloud Uploads`` or ``All cloud uploads``

File: _posts/development/2018-02-23-cloud-upload.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+<img src="/images/posts/cloud_upload_overview_2.png" alt="Cloud uploader" style="max-width: 800px;">
+
+<img src="/images/posts/cloud_upload_overview.png" alt="Cloud uploader" style="max-width: 800px;">
```
Comment (ChrisBr) [inline _posts/development/2018-02-23-cloud-upload.md]: What is the value of two screenshots except having the flash message on the second?

File: _posts/development/2018-02-23-cloud-upload.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+
+### Configuring Amazon EC2 credentials
+
+In order to be able to upload EC2 images from OBS to AWS you need to give OBS permissions,
+by creating a AWS role and adding the OBS account to it.
```
Comment (ChrisBr) [inline _posts/development/2018-02-23-cloud-upload.md]: Hm, it is a Amazon Resource Name (ARN). In general it is something like a role but I would use the official Amazon terms.

File: _posts/development/2018-02-23-cloud-upload.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+You can do this on the `Amazon EC2 configuration` page, where it is explained step by step.
```
Comment (ChrisBr) [inline _posts/development/2018-02-23-cloud-upload.md]: Maybe we should tell that we will create e.g. SSH key, VPC, VPC subnet, helper instance, volumes?

Comment (ChrisBr) [reply _posts/development/2018-02-23-cloud-upload.md]: Maybe we should also tell that if the user wants to revoke the permission, he just needs to delete the ARN in AWS (or just remove it from the configuration).

We could also link to the Amazon explanation for this, although this contains some technical details
https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html

File: _posts/development/2018-02-23-cloud-upload.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Cloud Upload
+category: development
+author: "OBS frontend team"
+---
+
+The upcoming release of OBS 2.9 will allow you to upload your EC2 images to [Amazon Web Services (AWS)](https://aws.amazon.com). :tada:
```
Comment (ChrisBr) [inline _posts/development/2018-02-23-cloud-upload.md]: I'm not sure if we should write this for the OBS 2.9 release but more for build.o.o. If we write it for OBS 2.9 we should also include that Administrators need to configure it first.

File: _posts/development/2018-02-23-cloud-upload.md (hunk 4 above)
Comment (ChrisBr) [reply _posts/development/2018-02-23-cloud-upload.md]: cloud_upload_aws.png image is not used

File: _posts/development/2018-02-23-cloud-upload.md (hunk 5 above)
Comment (Ana06) [reply _posts/development/2018-02-23-cloud-upload.md]: I agree, but 2.9 appeared in the card https://trello.com/c/KilbFTws/682-p5-3-29-cloud-upload-teaser :no_mouth:

File: _posts/development/2018-02-23-cloud-upload.md (hunk 2 above)
Comment (Ana06) [reply _posts/development/2018-02-23-cloud-upload.md]: the status is also different... but we can drop one :+1:

File: _posts/development/2018-02-23-cloud-upload.md (hunk 3 above)
Comment (Ana06) [reply _posts/development/2018-02-23-cloud-upload.md]: why don't we show this name in the steps that appear in OBS? :confused:

File: _posts/development/2018-02-23-cloud-upload.md (hunk 4 above)
Comment (Ana06) [reply _posts/development/2018-02-23-cloud-upload.md]: @ChrisBr 

> cloud_upload_aws.png image is not used

@bgeuken we forgot it! :disappointed_relieved:

File: _posts/development/2018-02-23-cloud-upload.md (hunk 3 above)
Comment (ChrisBr) [reply _posts/development/2018-02-23-cloud-upload.md]: We do, see at the bottom:
```
Please enter now the obtained Amazon Resouce Name (ARN):
```

File: _posts/development/2018-02-23-cloud-upload.md (hunk 5 above)
Comment (ChrisBr) [reply _posts/development/2018-02-23-cloud-upload.md]: I'm still not convinced! 
And as said, if you want to write it for 2.9 I would at least write that this does not work out of the box and the Administrator needs to set it up.

**Comment by Ana06** [comment]
@ChrisBr please re-review it

File: _posts/development/2019-01-01-cloud-upload.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+<img src="/images/posts/cloud_upload_uploader.png" alt="Cloud uploader" style="max-width: 800px;">
+
+If you haven't configured your account yet, you can use the link in the EC2 Uploader for the `Amazon EC2 configuration`.
+Check the [Configuring Amazon EC2 credentials section](#configuring-amazon-ec2-credentials) for more details.
+
+If your account is already configured, you can choose the AMI Name, the region you want to upload to and optionally the VPC Subnet ID.
+
+<img src="/images/posts/cloud_upload_uploader.png" alt="Cloud uploader" style="max-width: 800px;">
+
+By clicking `Upload`, the image process will start and you will get redirected to the `All cloud uploads` page:
```
Comment (ChrisBr) [inline _posts/development/2019-01-01-cloud-upload.md]: ```image process```?
I would call it upload process?

File: _posts/development/2019-01-01-cloud-upload.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+<img src="/images/posts/cloud_upload_overview.png" alt="Cloud uploader" style="max-width: 800px;">
+
+When it finishes, you will be able to see it in your Amazon account:
```
Comment (ChrisBr) [inline _posts/development/2019-01-01-cloud-upload.md]: You also see the AMI in the details column, maybe worth mentioning (but then you should maybe update the image of the table that it contains an AMI (maybe david has one))

File: _posts/development/2019-01-01-cloud-upload.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+<img src="/images/posts/cloud_upload_aws.png" alt="AWS" style="max-width: 800px;">
+
+
+### Configuring Amazon EC2 credentials
+
+In order to be able to upload EC2 images from OBS to AWS you need to give OBS permissions,
+by creating a [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) and adding the OBS account to it.
```
Comment (ChrisBr) [inline _posts/development/2019-01-01-cloud-upload.md]: ```s/a [Amazon Resource Name (ARN)]/an .../```

File: _posts/development/2019-01-01-cloud-upload.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+You can do this on the `Amazon EC2 configuration` page, where it is explained step by step. You can find more information about how to use an external ID when granting access to your AWS resources in [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
+
+Since this might incur costs, we recommend you to read [Amazon's price list](https://aws.amazon.com/de/partners/suse).
+
+<img src="/images/posts/cloud_upload_configuration.png" alt="Cloud upload configuration" style="max-width: 800px;">
+
+
+Try it out and enjoy uploading your images! :wink:
```
Comment (ChrisBr) [inline _posts/development/2019-01-01-cloud-upload.md]: ```uploading your images```
Maybe running your images in the :cloud: ?
(I don't enjoy the uploading, right? This is just the intermediate step to eventually run it in the cloud ...)

**Comment by Ana06** [comment]
@ChrisBr 

> You also see the AMI in the details column, maybe worth mentioning (but then you should maybe update the image of the table that it contains an AMI (maybe david has one))

It is broken at the moment, not possible to take an screenshot :sob: 

@DavidKang has this: https://user-images.githubusercontent.com/1212806/36740808-d0fe6afa-1be3-11e8-9020-c8a2d6eaffe7.png

can we use it? :thinking:

**Pull request merged**